AutoCollection/WebSnippet/snippet.js

```diff
@@ -0,0 +1,316 @@
+﻿(function (win, doc, snipConfig) {
+    var locn = win.location;
+    var helpLink = "https://go.microsoft.com/fwlink/?linkid=2128109";
+    var scriptText = "script";
+    var strInstrumentationKey = "instrumentationKey";
+    var strIngestionendpoint = "ingestionendpoint";
+    var strDisableExceptionTracking = "disableExceptionTracking";
+    var strAiDevice = "ai.device.";
+    var strAiOperationName = "ai.operation.name";
+    var strAiSdkVersion = "ai.internal.sdkVersion";
+    var strToLowerCase = "toLowerCase";
+    var strEmpty = "";
+    var strUndefined = "undefined";
+    var strCrossOrigin = "crossOrigin";
+
+    var strPostMethod = "POST";
+    var sdkInstanceName = "appInsightsSDK";         // required for Initialization to find the current instance
+    var aiName = snipConfig.name || "appInsights";  // provide non default instance name through snipConfig name value
+    if (snipConfig.name || win[sdkInstanceName]) {
+        // Only set if supplied or another name is defined to avoid polluting the global namespace
+        win[sdkInstanceName] = aiName;
+    }
+    var aisdk = win[aiName] || (function (aiConfig) {
+        var loadFailed = false;
+        var handled = false;
+        var appInsights = {
+            initialize: true,   // initialize sdk on download
+            queue: [],
+            sv: "4",            // Track the actual snippet version for reporting.
```

Comment:
This is v:"4" which has a bug when an exception occurs before the SDK is loaded -- grab the current one.